src/services/api/handlers/submit.ts

```diff
@@ -53,11 +54,11 @@ export const submit = async (event: APIGatewayEvent) => {
     });
   }
 
-  const activeSubmissionTypes = [
-    Authority.CHIP_SPA,
-    Authority.MED_SPA,
-    Authority["1915b"],
-    Authority["1915c"], // We accept amendments, renewals, and extensions for Cs
+  const activeSubmissionTypes: SeatoolAuthority[] = [
+    "chip spa",
+    "medicaid spa",
+    "1915(b)",
+    "1915(c)",
```

Comment:
Is this the check that prevents us from sending authority in a normal case? Can we update this to `Authority[]` (i.e. normal case authority values) and forget about converting it everywhere and getting mixed up? 

Am I making sense?